server/src/main/java/org/elasticsearch/snapshots/SnapshotInfo.java

```diff
@@ -51,6 +52,7 @@
 
     public static final String CONTEXT_MODE_PARAM = "context_mode";
     public static final String CONTEXT_MODE_SNAPSHOT = "SNAPSHOT";
+    public static final Version METADATA_FIELD_INTRODUCED = Version.V_8_0_0; // TODO Set this to the earliest version this is backported to
```

Comment:
This must be 8.0 until this gets backported, then needs to be changed after the backport is merged, otherwise the BWC tests fail.